test/JDBC/upgrade/latest/schedule

```diff
@@ -102,6 +102,7 @@ Test-Computed-Columns
 BABEL-1189
 BABEL-1243
 BABEL-1493
+BABEL-1876
```

Comment:
Latest is only for self MVU. This change has minor upgrade implications as well. Add in 14.5 schedule as well.

Reply:
Added